katalog/velero-on-prem/minio/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+namespace: kube-system
+
+resources:
+  - rbac.yaml
+  - init-job.yaml
+  - service.yaml
+  - sts.yaml
+
+vars:
+  - name: MINIO_NAMESPACE
+    objref:
+      kind: Service
+      name: minio
+      apiVersion: v1
+    fieldref:
+      fieldpath: metadata.namespace
+  - name: MINIO_SERVICE_NAME
+    objref:
+      kind: Service
+      name: minio
+      apiVersion: v1
+    fieldref:
+      fieldpath: metadata.name
+  - name: MINIO_STS_NAME
+    objref:
+      kind: StatefulSet
+      name: minio
+      apiVersion: apps/v1
+    fieldref:
+      fieldpath: metadata.name
+  - name: MINIO_SETUP_JOB_NAME
+    objref:
+      kind: Job
+      name: minio-setup
+      apiVersion: batch/v1
+    fieldref:
+      fieldpath: metadata.name
+
+images:
+  - name: minio/minio
+    newTag: RELEASE.2020-01-25T02-50-51Z
+  - name: minio/mc
+    newTag: RELEASE.2020-01-25T03-02-19Z
+
+generatorOptions:
+  disableNameSuffixHash: true
```

Comment:
Because we use the same `base` both for onprem and clouds. As we generate `cloud-credentials` using kustomize only when deploying on prem, we need to set this feature.
Cloud `cloud-credentials` has to be created from the output of terraform modules and they does not contain suffixes.

You can see an example in the pipeline:

```
$ terraform output cloud_credentials > /shared/cloud_credentials.config
$ kubectl create secret generic cloud-credentials --from-file=cloud=/shared/cloud_credentials.config --dry-run -o yaml | kubectl apply -f - -n kube-system
```

I hope i explained it well. 

Thanks!

Reply:
Perfect.